lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,16 @@
 
 def palindrome_permutation?(string)
-  
+  hash_table = {}
+
+  string.each_char do |c|
+    hash_table[c] ? hash_table[c] += 1 : hash_table[c] = 1
+  end
+
+  odd_numbered_letters = 0
```

Comment:
I like that this describes the number! Could also indicate in the variable name that it's counting something e.g. `odd_numbered_letters_count`.